docs/CHANGELOG.asciidoc

```diff
@@ -59,6 +59,15 @@ To review the breaking changes in previous versions, refer to the following:
 Alerting::
 Fixes an issue where alerting rules that were created or edited in 8.2.0 stopped running when you upgraded {kib} to 8.3.0 or 8.3.1 {kibana-pull}135663[#135663]
 
+====
```

Comment:
Can we merge the information in this PR with https://github.com/elastic/kibana/pull/135876?